### ISSUE TITLE
[BUG] Grace currently does not affect Prayer of Mending (and especially the first tick)

### ISSUE THREAD

**Issue by Pozzel** [JamminL/cata-classic-bugs#1182]
#### 1.2. How to Reproduce
           1. Use prayer of mending on a target without grace (u can use it on yourself and proc it with shadow word death on a dummy)
           2. Replicate it with grace being active on you (1/2/3 stacks). The numbers should increase, the more stacks of grace u have on the target.  Right now, they don't.
           
  #### 1.3  Source Material
U can see PoM being used frequently and it here. U can also see, that the first tick of PoM heals alot more, than it currently does on the beta. U can see the priests hitting up to 30k crits with the first tick of Prayer of Mending. I wrote it down for the first tick of Prayer of Mending (with the glyph). The exact timestamps for the first video are as following. In addition, I wrote down the exact buffs and how many grace stacks were attached to the target. 
00:50 | 01:02 | 01:15 | 01:31 | 01:42 | 02:11 | 02:43 | 02:16
Power torrent int | 0 | 0 | 400 | 0 | 0 | 0 | 0 | 0
Mage buff | 1 | 1 | 1 | 1 | 1 | 1 | 1 | 1
Grace | 3 | 3 | 2 | 0 | 3 | 2 | 0 | 0
Quickening | 1 | 1 | 1 | 1,2 | 1 | 1 | 1,2 | 1
POM non crit | 14781 | 29563 (crit) | 13461 | 19516 (crit)  | 14782 | 13461 | 9758 | 9758
Prayer of mending ticks after the first tick: 5.836 

The exact timestamps for the second video, where u can see PoM healing on the first tick, are:
04:18 | 05:30 | 09:42 | 10:00 | 19:30 | 26:10 | 31:04 |38:40 | 39:06 | 39:25 | 40:10 | 40:48 | 42:28 | 42:52 | 43:14 | 43:34 | 44:01 | 44:34 | 
00:31 | 00:58 | 01:50
Grace multiplier | 1 | 1,24 | 1 | 1 | 1,16 | 1,24 | 1,16 | 1 | 1,24 | 1,24 | 1,24 | 1 | 1,16 | 1 | 1,24 | 1,24 | 1 | 1 | 1,24 | 1,24 | 1
Spellpower from buffs etc | 0 | 0 | 400 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 0 | 420 | 0 | 0 | 420 | 0 | 0
Healing: 9827 | 14780 | 10477 | 10131 | 13308 | 26181 (crit) | 13090 | 10046 | 30221 (crit) | 15656 | 30221 (crit) | 10046 | 13308 | 10046 | 30221 (crit) | 31311 (crit) | 20092 (crit) | 10046 | 15300 | 14780 | 9827

https://www.youtube.com/watch?v=2QO22zFAalM&t=1185s

#### 2.1. Grace should increase prayer of mendings healing on the target. As u can see in the videos, the first tick of prayer of mending was particularly affected by that. Which are all from the tournament realm 12 years ago, where people had to use Season 9 gear but was played on the 4.3.4 Patch.
[POM_maths (2).csv](https://github.com/JamminL/cata-classic-bugs/files/15246840/POM_maths.2.csv)

**Comment by AidanZMoon** [comment]
The base healing and spell power coefficient of Prayer of Mending are correct to 4.3.4.
Grace not working is incorrect.
The reason you're seeing such a disparity between current PoM on 4.4.0 is because Glyph + Grace had a really intense interaction back in the day.
You can even see it with your own research - you have 9758 on 0 stack Grace first hit PoM. and then 14781 on 3 stack Grace first hit PoM. That means Grace is giving about a 51% increase when paired with the Glyph. Grace was not giving a 24% increase in 4.3.4 to PoM. It looks like it was giving way more, and that lines up with my own testing.

TL;DR the spell power coefficients and base healing of PoM are correct to 4.3.4.
Grace is not working correctly though, and Grace was giving a ton of healing to PoM in 4.3.4, so you are noticing the discrepancy between the two, but attributing it to PoM coefficient or base, and that is not correct. It's just Grace that is not working, but when it works, it works better than it should've when paired with the Glyph.
